diff --git a/node_modules/react-native-sortable-list/.DS_Store b/node_modules/react-native-sortable-list/.DS_Store
new file mode 100644
index 0000000..6fc2086
Binary files /dev/null and b/node_modules/react-native-sortable-list/.DS_Store differ
diff --git a/node_modules/react-native-sortable-list/src/Row.js b/node_modules/react-native-sortable-list/src/Row.js
index 32718a8..e35fbce 100644
--- a/node_modules/react-native-sortable-list/src/Row.js
+++ b/node_modules/react-native-sortable-list/src/Row.js
@@ -9,7 +9,7 @@ export default class Row extends Component {
     animated: PropTypes.bool,
     disabled: PropTypes.bool,
     horizontal: PropTypes.bool,
-    style: ViewPropTypes.style,
+    style: ViewPropTypes?.style??{},
     location: PropTypes.shape({
       x: PropTypes.number,
       y: PropTypes.number,
diff --git a/node_modules/react-native-sortable-list/src/SortableList.js b/node_modules/react-native-sortable-list/src/SortableList.js
index 78cd576..4c9c095 100644
--- a/node_modules/react-native-sortable-list/src/SortableList.js
+++ b/node_modules/react-native-sortable-list/src/SortableList.js
@@ -17,9 +17,9 @@ export default class SortableList extends Component {
   static propTypes = {
     data: PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired,
     order: PropTypes.arrayOf(PropTypes.any),
-    style: ViewPropTypes.style,
-    contentContainerStyle: ViewPropTypes.style,
-    innerContainerStyle: ViewPropTypes.style,
+    style: ViewPropTypes?.style??{},
+    contentContainerStyle: ViewPropTypes?.style??{},
+    innerContainerStyle: ViewPropTypes?.style??{},
     sortingEnabled: PropTypes.bool,
     scrollEnabled: PropTypes.bool,
     horizontal: PropTypes.bool,
